### DISCUSSION TITLE
Grid Magnetic Angle: [Mapper Grivation]

### DISCUSSION THREAD

**Discussion by PskovFF** [OpenOrienteering/mapper#2128] [Questions & Answers] [answered]
Hi.
Why when i use Grid Magnetic Angle: [Mapper Grivation], OOM using Magnetic Declination without Grid Convergence?

Images to example:
![PUn8Yplsw-Q - копия (2)](https://user-images.githubusercontent.com/122890459/212920392-e4f25a8f-11a3-45cf-a4df-62f31f322ea3.jpg)
![PUn8Yplsw-Q - копия (3)](https://user-images.githubusercontent.com/122890459/212920418-da5c2d21-95ac-40b9-a28b-a44b7fea6396.jpg)
![PUn8Yplsw-Q - копия (4)](https://user-images.githubusercontent.com/122890459/212920433-e74bc7e1-10b5-4478-96ca-e5bd45907323.jpg)
![PUn8Yplsw-Q - копия (5)](https://user-images.githubusercontent.com/122890459/212920453-e7c5662a-65df-4ebe-b679-390e6674b699.jpg)
Sorry for my english :)

**Comment by pkturner** [comment, marked as answer]
The map (14).osm template uses a different projection which has a convergence of 0°. 
Mapper always has magnetic north vertical. For that reason, the OpenStreetMap features must appear rotated by the declination (10.6°).

**Comment by PskovFF** [comment]
Oh, its really is. Thanks.
